src/main/java/com/exasol/adapter/sql/SqlNodeVisitor.java

```diff
@@ -14,76 +14,76 @@
  */
 public interface SqlNodeVisitor<R> {
 
-    public R visit(SqlStatementSelect select) throws AdapterException;
+    R visit(SqlStatementSelect select) throws AdapterException;
```

Comment:
Keep public, it's cleaner. Check if there are IDE rules that mark public as an error (which it isn't). Deactivate these rules.

Reply:
done